server/template.ts

```diff
@@ -3,7 +3,7 @@ import { urlParamsNames } from "./params.ts";
 import { flagsNormalization } from "./dictionary.ts";
 
 const style = await Deno.readTextFile(
-  new URL("../static/main.css", import.meta.url)
+  new URL("../static/main.css", import.meta.url),
```

Comment:
Are these changes generated by `deno fmt`? (I'm _very_ new to Deno: I know that it has a formatter, but I never used it)

Reply:
Yes, this changes are generated by `deno fmt`